tests/test_workspace.py

```diff
@@ -25,11 +25,13 @@ class TestCreate:
     def test_success(self, requests_mock, cli_runner):
         name = "TestName"
         new_id = "2"
+        queues = None
```

Comment:
Consider leaving the initialization of this variable out and just put `None` into the `queues` below, as it is not going to be used anywhere else in the code of this test.

Reply:
yes, you are right